### PR TITLE
Pinning icon alignment 

### LINKED ISSUE #1728: Pinning icon is not well aligned.
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/master/CONTRIBUTING.md) for onboarding!
___
Pin icon on the sidebar is not aligned correctly.  Suspected PR https://github.com/Expensify/Expensify.cash/pull/1721

## Action Performed:
 Pina chat and see the Sidebar

## Platform:
Where is this issue occurring?

 Web
 
image
#1728 
cc @roryabraham

### PR DESCRIPTION
**Pull request by aliabbasmalik8**
@roryabraham Please have a look
### Details
Fixed regression that caused due to my PR 
### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/1728
https://github.com/Expensify/Expensify.cash/issues/1661

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [ ] Android

#### Web
image

#### Mobile Web
image

#### Desktop
image

#### iOS
image

### CODE REVIEW

**Review by roryabraham — CHANGES_REQUESTED** [review]

File: src/styles/styles.js [hunk 1]
```diff
@@ -509,6 +509,10 @@ const styles = {
         paddingLeft: 20,
     },
 
+    iconsRow: {
```
[the remark this answers is not in the document]
Comment (aliabbasmalik8) [reply src/styles/styles.js:512]: Okay, on it

[the remark this answers is not in the document]
Comment (aliabbasmalik8) [reply src/styles/styles.js]: Updated PR according to your feedback.
Thanks

**Review by roryabraham — APPROVED** [review]
LGTM, thanks for fixing

**Review by Dal-Papa — APPROVED** [review]

**Comment by shawnborton** [comment]
Hmm actually I think this did not fix the issue. 

You can see from the outline that the entire row is not clickable:
image

The outline should encompass the entire row, and the pin & pencil icons should be nested inside of the row and they should be clickable.

**Pull request merged**